xml/System.Windows.Forms/LinkClickedEventArgs.xml

```diff
@@ -1,206 +1,208 @@
+        </Attribute>
+      </Attributes>
+      <Parameters>
+        <Parameter Name="linkText" Type="System.String" />
+      </Parameters>
+      <Docs>
+        <param name="linkText">The text of the link that is clicked in the <see cref="T:System.Windows.Forms.RichTextBox" /> control.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Windows.Forms.LinkClickedEventArgs" /> class.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The text specified in the `linkText` parameter can be a URL, file path, or other data that you want specified for the link being clicked.  
+  
+ ]]></format>
+        </remarks>
+      </Docs>
+    </Member>
+    <Member MemberName=".ctor">
+      <MemberSignature Language="C#" Value="public LinkClickedEventArgs (string? linkText, int linkStart, int linkLength);" />
+      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string linkText, int32 linkStart, int32 linkLength) cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.Windows.Forms.LinkClickedEventArgs.#ctor(System.String,System.Int32,System.Int32)" />
+      <MemberSignature Language="VB.NET" Value="Public Sub New (linkText As String, linkStart As Integer, linkLength As Integer)" />
+      <MemberSignature Language="F#" Value="new System.Windows.Forms.LinkClickedEventArgs : string * int * int -&gt; System.Windows.Forms.LinkClickedEventArgs" Usage="new System.Windows.Forms.LinkClickedEventArgs (linkText, linkStart, linkLength)" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; LinkClickedEventArgs(System::String ^ linkText, int linkStart, int linkLength);" />
+      <MemberType>Constructor</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Windows.Forms</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Parameters>
+        <Parameter Name="linkText" Type="System.String" Index="0" FrameworkAlternate="net-6.0" />
+        <Parameter Name="linkStart" Type="System.Int32" Index="1" FrameworkAlternate="net-6.0" />
+        <Parameter Name="linkLength" Type="System.Int32" Index="2" FrameworkAlternate="net-6.0" />
+      </Parameters>
+      <Docs>
+        <param name="linkText">The text of the link being clicked.</param>
+        <param name="linkStart">The start of the link span being clicked.</param>
+        <param name="linkLength">The length of the link span being clicked.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Windows.Forms.LinkClickedEventArgs" /> class.</summary>
+        <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentOutOfRangeException">The value for <paramref name="linkStart" /> or <paramref name="linkLength" /> is negative.
+-or- The values for <paramref name="linkStart" /> and <paramref name="linkLength" /> would overflow addition.</exception>

```

Comment:
I need to fix the bug in the tool. It's not adding the proper endlines that wrap an `-or-`:

```suggestion
        <exception cref="T:System.ArgumentOutOfRangeException">The value for <paramref name="linkStart" /> or <paramref name="linkLength" /> is negative.

-or-

The values for <paramref name="linkStart" /> and <paramref name="linkLength" /> would overflow addition.</exception>

```